owlbot.py

```diff
@@ -21,75 +21,60 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICBazel()
 common = gcp.CommonTemplates()
 
-# ----------------------------------------------------------------------------
-# Generate Grafeas GAPIC layer
-# ----------------------------------------------------------------------------
-library = gapic.py_library(
-    service="grafeas",
-    version="v1",
-    bazel_target="//grafeas/v1:grafeas-v1-py",
-    proto_output_path="grafeas/grafeas_v1/proto",
-    include_protos=True,
-)
+default_version = "v1"
 
-excludes = ["README.rst", "setup.py", "docs/index.rst"]
+for library in s.get_staging_dirs(default_version):
+    # Make package name 'grafeas'
+    s.replace(library / "grafeas/**/*.py", "grafeas-grafeas", "grafeas")
 
-s.move(library, excludes=excludes)
+    # ----------------------------------------------------------------------------
+    # Remove google-specific portions of library
+    # ----------------------------------------------------------------------------
 
+    # Please see this PR for more context
+    # https://github.com/googleapis/google-cloud-python/pull/8186/
 
-# Make package name 'grafeas'
-s.replace("grafeas/**/*.py", "grafeas-grafeas", "grafeas")
+    # Remove default service address, default scopes, default credentials
+    # Users must pass a transport to the client constructor
 
-# ----------------------------------------------------------------------------
-# Remove google-specific portions of library
-# ----------------------------------------------------------------------------
+    # Remove default endpoint
+#     s.replace(
+#     library / "grafeas/**/*client.py",
+#     r"""\s+DEFAULT_ENDPOINT\s+=\s+"containeranalysis\.googleapis\.com"
+# \s+DEFAULT_MTLS_ENDPOINT\s+=\s+_get_default_mtls_endpoint\.__func__\(  # type: ignore
+# \s+DEFAULT_ENDPOINT
+# \s+\)""",
+#     "",
+#     )
```

Comment:
I commented out this line because I'm unsure if it is intended. This doesn't reflect the state of the code in master : https://github.com/googleapis/python-grafeas/blob/master/grafeas/grafeas_v1/services/grafeas/client.py#L131

Reply:
The replacement needs to be fixed to account for the new code surface

https://github.com/googleapis/python-grafeas/blob/b8747ec90f486708146d3c2c25cff44fc58fa803/synth.py#L50-L54

Reply:
Done.